### PR TITLE
[#109515840] Add readme (and 2)

### PR DESCRIPTION
**Pull request by keymon**
# What

Update README formating

# Details

Basic information about how pipeline operates operates and how to set-up your environment to be able to deploy your DEV Cloudfoundry instance.

# How to test

Proofreading is highly welcome. Also, please follow `USAGE` section to check whether instructions are correct.

# Who can test 
Anyone but @combor and @jimconner   or @keymon

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,15 +24,42 @@ export TF_VAR_AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
...
+
+3. Login on the local concourse:
+
+     ```
+fly login --concourse-url http://192.168.100.4:8080 sync
+```
+
+### Deploy the deployer concourse
+
+`./concourse/scripts/create-deployer.sh`
```
Comment (dcarley) [inline README.md]: Could you change this to triple backticks? Change was lost in https://github.com/alphagov/paas-cf/pull/6#discussion_r47354327

File: README.md [hunk 2]
```diff
@@ -34,15 +24,42 @@ export TF_VAR_AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
...
+### Destroy the deployer concourse
 
-- `git clone https://github.com/alphagov/paas-cf.git`
-- `cd paas-cf`
-- `vagrant up`
-- ```sudo curl http://192.168.100.4:8080/api/v1/cli?arch=amd64&platform=`uname | tr '[:upper:]' '[:lower:]'` -o /usr/local/bin/fly```
-- `sudo chmod +x /usr/local/bin/fly`
-- `fly login --concourse-url http://192.168.100.4:8080 sync`
-- `./concourse/scripts/create-deployer.sh`
+`./concourse/scripts/destroy-deployer.sh`
```
Comment (dcarley) [inline README.md]: Could you change this to triple backticks? Change was lost in https://github.com/alphagov/paas-cf/pull/6#discussion_r47354321

File: README.md (hunk 1 above)
Comment (combor) [inline README.md]: Done.

File: README.md (hunk 2 above)
Comment (combor) [inline README.md]: Done.

**Comment by mtekel** [comment]
Please add information about using only 2 spaces for TF intendation...

**Comment by dcarley** [comment]
:+1:

**Pull request merged**